### PR TITLE
Another pass at dotnet cli

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,19 @@
 language: csharp
 
+dist: trusty
```
Comment (chrisdunelm) [inline .travis.yml]: According to: https://docs.travis-ci.com/user/trusty-ci-environment/
You're also meant to have "sudo: required".

File: myget.bat [hunk 2]
```diff
@@ -1,31 +1,24 @@
...
+  dotnet build --configuration %Configuration% %%P || goto error
 )
 
 for %%P in (%TESTS%) do (
   echo Testing %%P
-  call dnu build --configuration %Configuration% %%P
-  IF ERRORLEVEL 1 goto error
-  dnx --configuration %Configuration% -p %%P test
-  IF ERRORLEVEL 1 goto error
+  dotnet build --configuration %Configuration% %%P || goto error
```
Comment (chrisdunelm) [inline myget.bat]: This isn't strictly required, as "test" will also build.

File: .travis.yml (hunk 1 above)
Comment (jskeet) [inline .travis.yml]: Have never needed that... it does say "All Trusty images have sudo enabled" so maybe that explains why it's working? I'm happy to modify it if you'd like, but it does seem to be okay...

File: myget.bat (hunk 2 above)
Comment (chrisdunelm) [inline myget.bat]: (This is a nit, I'm happy either way...)

Comment (jskeet) [inline myget.bat]: True. I previously wanted to separate them to make it clear which bit had failed, but I'm happy to embrace a smaller build script.

File: .travis.yml (hunk 1 above)
Comment (chrisdunelm) [inline .travis.yml]: It's fine, leave it as it is

**Comment by chrisdunelm** [comment]
LGTM :)

File: myget.bat (hunk 2 above)
Comment (jskeet) [inline myget.bat]: Humbug - intended to remove the build step, but failed on the commit amend. Will leave it for now.

**Pull request merged**